src/app/index.js

```diff
@@ -0,0 +1 @@
+export {default} from './app';
```

Comment:
I had to sleep on it... I'm not sure if I like it. This file doesn't serve a huge purpuse right? Would it not be better to move the contents of `./app`, `./orgs` etc into their `index.js` file? Or maybe import them differently later on?

Reply:
This is from gds-way.... I didn't like it at first ... but actually it grew on me.

* it's quite nice to be able to treat the directories like mini npm packages and index.js is the 'public' api for it
* It's nice to be able to reference the directory in imports `import app from './app';` rather than `import app from './app/app';`

Reply:
Yeah, I agree with the "clean" approach in imports. What if the `app.js` was actually named `index.js` 🤔 We should make sure it only contains "routes" and no logic though.